packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-apis/async-component-access-api-prop-04.input.tsx

```diff
@@ -0,0 +1,10 @@
+// If it's already awaited, don't wrap it again
```

Comment:
doesn't seem like that's what's happening here, unless I'm misunderstanding the comment.

Reply:
it was a todo comment during dev, will remove